ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -213,13 +214,22 @@ public override void OnDropViewInstance(ThemedReactContext reactContext, Border
         /// <returns>The image view instance.</returns>
         protected override Border CreateViewInstance(ThemedReactContext reactContext)
         {
-            return new Border
+            Border border = new Border
             {
                 Background = new ImageBrush
                 {
                     Stretch = Stretch.UniformToFill,
                 },
             };
+
+            // Setting AutomationProperties.Name to some string and then clearing it will guarantee that
```

Comment:
>// [](start = 12, length = 2)

maybe encapsulate it in AccessibilityHelpers.OnViewInstanceCreated()?

Reply:
or maybe put this into framework caller of the CreateViewInstance

---
In reply to: [169484625](https://github.com/Microsoft/react-native-windows/pull/1645#discussion_r169484625) [](ancestors = 169484625)

Reply:
Done

---
In reply to: [169485221](https://github.com/Microsoft/react-native-windows/pull/1645#discussion_r169485221) [](ancestors = 169485221,169484625)